src/services/color/color.service.ts

```diff
@@ -91,7 +83,12 @@ export class ColorService {
     }
 
     getColor(color: ColorIdentifier): ThemeColor {
```

Comment:
Change ColorIdentifier to string and remove the definition below.

Reply:
This has been documented in UX-Aspects so could cause a breaking change if we remove it

Reply:
You can change the type to string without breaking anything. String is a superset of ColorIdentifier.

Reply:
Yes I changed the type, we just cant remove the definition